lib/cretonne/src/isa/intel/abi.rs

```diff
@@ -168,6 +169,31 @@ pub fn callee_saved_registers(flags: &shared_settings::Flags) -> &'static [RU] {
     }
 }
 
+fn callee_saved_registers_used(
+    flags: &shared_settings::Flags,
+    func: &ir::Function,
+) -> AllocatableSet {
+    let mut all_callee_saved = AllocatableSet::empty();
+    for reg in callee_saved_registers(flags) {
+        all_callee_saved.free(GPR, *reg as RegUnit);
```

Comment:
While we don't support the Windows x64 calling convention yet, it has some XMM registers as callee-saved. What would you think about renaming `callee_saved_registers` to `callee_saved_gprs`, to remind us to do something else when we need to support callee-saved xmm registers?

Also, it's not new in your patch here, but `callee_saved_registers` doesn't need to be `pub`.